journeyman/webdev/markdown.md

```diff
@@ -0,0 +1,34 @@
+# Markdown 
+
+## Intro
+
+Markdown is a text-to-html tool for web developers. It allows you to write information that is easy to write, and easy to read, than it converts it to an HTML structure for you. 
+
+- Markdown has many uses some of which are: 
+    - It works great for ReadMes and documentation.
+    - You can write long documents with it.
+    - You can use it to format your text in a lot of popular modern chat apps, like Discord and Slack.
+    - You can use it in Github comments, issues, and pull requests (In fact, Github has it's own special flavor of Markdown).
+
+## Suggested Learning
+
+- Follow this [Markdown tutorial](http://www.markdowntutorial.com/) for hands on learning.
+- View the different .md files in this Webdev repository.
+
+## Requirements
+
+- Create a markdown file containing whatever you want.
+- It must include: 
+    - A header
+    - A description
+    - A link
+    - An image
+- You can do this on an online markdown editor. A good one to use is [Markdown Editor.](https://jbt.github.io/markdown-editor/)
+- Share your file (or a screenshot of it if you decided to use the online editor) with your mentor to have them pass it off.
+
+## Extra Learning
+
+- Watch [this video](https://www.youtube.com/watch?v=_gknWWa2OF0) which is helpfull for visual learners.
+- Read the Intro and Syntax on [Markdown](https://daringfireball.net/projects/markdown/)
```

Comment:
Leave the Video on Extra Learning, but put the intro and syntax link back in the suggested learning.